pkg/agent/datapath/localBridge.go

```diff
@@ -285,6 +285,219 @@ func (l *LocalBridge) BridgeInit() {
 	if err := l.initFromLocalRedirectTable(sw); err != nil {
 		log.Fatalf("Failed to init local bridge from local redirect table, error: %v", err)
 	}
+	if l.datapathManager.AgentInfo.EnableCNI {
+		if err := l.initCniRelatedFlow(sw); err != nil {
+			log.Fatalf("Failed to init cni related flows, error: %v", err)
+		}
+	}
+}
+
+func (l *LocalBridge) initLocalGwArpFlow(sw *ofctrl.OFSwitch) error {
+	// arp response flow
+
+	// target for local pod
+	arpPodFlow, _ := l.vlanInputTable.NewFlow(ofctrl.FlowMatch{
+		Priority:   HIGH_MATCH_FLOW_PRIORITY,
+		InputPort:  uint32(LOCAL_GATEWAY_PORT),
+		Ethertype:  PROTOCOL_ARP,
+		ArpTpa:     &l.datapathManager.AgentInfo.PodCIDR[0].IP,
+		ArpTpaMask: (*net.IP)(&l.datapathManager.AgentInfo.PodCIDR[0].Mask),
+	})
+	flood, _ := sw.OutputPort(openflow13.P_FLOOD)
+	if err := arpPodFlow.Next(flood); err != nil {
+		return fmt.Errorf("failed to install flow, error: %v", err)
+	}
+
+	// target for other ip, response arp with uplink gateway mac address
+	arpGwFlow, _ := l.vlanInputTable.NewFlow(ofctrl.FlowMatch{
+		Priority:  MID_MATCH_FLOW_PRIORITY,
+		InputPort: uint32(LOCAL_GATEWAY_PORT),
+		Ethertype: PROTOCOL_ARP,
+	})
+	// set actions for arp response
+	if err := arpGwFlow.MoveField(32, 0, 0, "nxm_of_arp_tpa", "nxm_of_arp_spa", false); err != nil {
+		return err
+	}
+	if err := arpGwFlow.LoadField("nxm_nx_arp_sha", ParseMacToUint64(l.datapathManager.AgentInfo.GatewayMac), openflow13.NewNXRange(0, 47)); err != nil {
+		return err
+	}
+	if err := arpGwFlow.MoveField(48, 0, 0, "nxm_nx_arp_sha", "nxm_nx_arp_tha", false); err != nil {
+		return err
+	}
+	if err := arpGwFlow.LoadField("nxm_of_arp_op", 0x0002, openflow13.NewNXRange(0, 15)); err != nil {
+		return err
+	}
+	fakeMac, _ := net.ParseMAC("77:77:77:77:77:77")
```

Comment:
please refer 
![image](https://user-images.githubusercontent.com/88091134/141036603-6697ead6-46f5-4bf5-9dfd-36f3bb5a468d.png): such as "e2:xx:xx:xx:xx:xx", and make this as const

Reply:
LAA mac address has such formats
x2‑xx‑xx‑xx‑xx‑xx
x6‑xx‑xx‑xx‑xx‑xx
xA‑xx‑xx‑xx‑xx‑xx
xE‑xx‑xx‑xx‑xx‑xx
choose ee:ee:ee:ee:ee:ee here.